client/my-sites/site-settings/analytics/form-cloudflare-analytics.js

```diff
@@ -235,7 +235,7 @@ const mapStateToProps = ( state ) => {
 	const siteId = getSelectedSiteId( state );
 	const isAnalyticsEligible = siteHasFeature( state, siteId, FEATURE_GOOGLE_ANALYTICS );
 	const siteIsJetpack = isJetpackSite( state, siteId );
-	const sitePlugins = site ? getPlugins( state, [ site.ID ] ) : [];
+	const sitePlugins = site ? getFilteredAndSortedPlugins( state, [ site.ID ] ) : [];
```

Comment:
I cannot find a place that this `sitePlugins` prop is used in order to test this switch, but I'm not confident enough to remove it. If I'm missing something here please let me know.